### PR TITLE
Update Post-Processing-Tonemapping.md

### PR DESCRIPTION
**Pull request by emilybrown1**
In response to https://jira.unity3d.com/browse/DOCF-986

Added paragraph under first sentence and a new description for Neutral mode

### CODE REVIEW

**Review by ValGrimm-U3D — APPROVED** [review]
Looks good to me.
Approving https://jira.unity3d.com/browse/DOCF-996
Added Francesco for approval from someone on the dev team so this can get published.

File: com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Tonemapping.md [hunk 1]
```diff
@@ -17,7 +19,7 @@ Tonemapping is the process of remapping HDR values of an image in a range suitab
...
+| **Mode**              | Use the drop-down to select a tonemapping algorithm to use for color grading. The options are:<br />&#8226; **None**: Use this option if you do not want to apply tonemapping.<br />&#8226; **Neutral**: Neutral mode is quite configurable. It is especially suitable for situations where you do not want the tone mapper to color grade your content.<br />&#8226; **ACES**: Use this option to apply a close approximation of the reference ACES tonemapper for a more filmic look. It is more contrasted than Neutral and has an effect on actual color hue & saturation. Note that if you use this tonemapper all the grading operations will be done in the ACES color spaces for optimal precision and results.<br />&#8226; **Custom**: Use this option if you want to specify the tonemapping settings yourself. Selecting this mode exposes properties that allow you to customize the tonemapping curve.<br />&#8226; **External**: Use this option if you want to specify your own lookup table. |
```
[the remark this answers is not in the document]
Comment (emilybrown1) [reply com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Tonemapping.md:22]: @ValGrimm-U3D Do you know who wrote/asked for this description? It was given to me as a direct quote in the ticket.

[the remark this answers is not in the document]
Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Tonemapping.md:22]: This is only the case for HDR so I assume the info comes from there? I have no idea where it comes from.

[the remark this answers is not in the document]
Comment (ValGrimm-U3D) [reply com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Tonemapping.md:22]: Yes, it did. I understood that the description of the neutral tonemapping mode in the HDR doc was more generally applicable and I should make the main tonemapping page consistent with it? It seems that was a misunderstanding. We can cancel this PR if we do not need to adjust the tonemapping page.

**Comment by emilybrown1** [comment]
Closed as this change isn't actually needed

**Pull request closed without merging**